### ISSUE TITLE
[Sourcing-2][Accessibility: ACC-270.6] List Item: UI cannot be activated with keyboard.

### ISSUE THREAD

**Issue by JyotiKumari1507** [SAP/fundamental-ngx#11292]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
Component having issue: https://sap.github.io/fundamental-ngx/#/core/list
The items within the list, cannot be activated using keyboard.
It must be possible to invoke/activate each & every interactive UI, using the keyboard. 

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All latest version

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Use any list item. check on select of items within the list, isn't activated using keyboard.

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations. 
Yes we have got this bug from Accessibility expert only.

#### Did you check the documentation and the API?
Yes

**Comment by droshev** [comment]
@JyotiKumari1507 The navigation list is fully functional with the keyboard. Can you provide a stackblitz example?

**Comment by divyamalhotra488** [comment]
@droshev Please find video recording for the issue: [here](https://sap-my.sharepoint.com/:v:/p/divya01/EYdWDapZ93BPpw0hvRkxctYB3AR8AqqSkqaevTdMtBRH9A?nav=eyJyZWZlcnJhbEluZm8iOnsicmVmZXJyYWxBcHAiOiJPbmVEcml2ZUZvckJ1c2luZXNzIiwicmVmZXJyYWxBcHBQbGF0Zm9ybSI6IldlYiIsInJlZmVycmFsTW9kZSI6InZpZXciLCJyZWZlcnJhbFZpZXciOiJNeUZpbGVzTGlua0NvcHkifX0&e=O62h3F)

CC: @JyotiKumari1507

**Comment by droshev** [comment]
@JyotiKumari1507 did you set all aria attributes needed? If so, can you create a small stackblitz example demonstrating that you cannot set this on the application side? Did you use the navigation list?

**Comment by JyotiKumari1507** [comment]
@divyamalhotra488 please check on this from our side and share stackblitz example

**Comment by droshev** [comment]
@JyotiKumari1507 We didn't get updates for some time so we are closing the issue. If you think there is a bug, please, create a stackblitz example, document the steps and what the expectation is in a new ticket.

**Comment by divyamalhotra488** [comment]
Hi @droshev, raised new defect https://github.com/SAP/fundamental-ngx/issues/11963
